### PR TITLE
[Rollout] Production Rollout Nov 11, 2020

### PR DESCRIPTION
**Pull request by sunandabalu**

### CODE REVIEW

File: src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs [hunk 1]
```diff
@@ -464,20 +470,31 @@ private void CreateOrUpdateDisabledSourcesBlock(XmlDocument nugetConfig, Diction
                 // If there's a clear node in the children of the disabledSources, we want to put any of our entries after the last one seen.
                 if (disabledSourcesNode.HasChildNodes)
                 {
-                    for (int i = 0; i < disabledSourcesNode.ChildNodes.Count; i++)
+                    XmlNode currentNode = disabledSourcesNode.FirstChild;
+
+                    while (currentNode != null)
                     {
-                        if (disabledSourcesNode.ChildNodes[i].Name.Equals("clear", StringComparison.InvariantCultureIgnoreCase))
+                        if (currentNode.Name.Equals("clear", StringComparison.InvariantCultureIgnoreCase))
```
Comment (MattGal) [inline src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs:477]: I feel pretty confident about this change (has test coverage in literally every test, is a small change) but I did merge it this morning, is that a problem?

**Review by MattGal — APPROVED** [review]
Signing off but note my comment about my recently-merged change.

File: src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs (hunk 1 above)
Comment (sunandabalu) [reply src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs:477]: so a green build should mean this change is tested?

Comment (MattGal) [reply src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs:477]: Yes; I made sure that without the fixes the tests failed (by adding the double-call) and that no expected inputs or outputs had to be changed.  It's just that I checked in recently, so I'll let you make that call.

Comment (sunandabalu) [reply src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs:477]: Perfect, then there should be no issues in taking this change.

**Review by riarenas — APPROVED** [review]

**Pull request merged**